source/index.html

```diff
@@ -2,11 +2,160 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="../source/img/icon/favicon.png">
+    <link rel="stylesheet" href="../source/css/">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <h1 class="visually hidden">CAT ENERGY</h1>
+    <header class="main__header">
+      <a class="header__logo logo__link" href="#">
+        <img class="logo-link__img" width="33" height="37" alt="Логотип с котом." src="../source/img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main__link" href="#">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип магазина" src="../source/img/icon/logo_main.svg">
```

Comment:
Если есть текст на картинке, то его стоит зачитать
```suggestion
        <img class="logo-main__img" width="101" height="18" alt="Логотип «Кэт Энерджи»." src="img/icon/logo_main.svg">
```